wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,105 @@
+
+## Gameplay
+
+These servers are focused on giving users a place to discuss osu!'s main attraction — the gameplay! They serve as hubs for players to interact with their daily gaming ventures in osu!.
+
+| Name | Owner | Description |
+| :-- | :-- | :-- |
+| [osu! Game](https://discord.gg/osu) | [oink](https://osu.ppy.sh/users/300173) | **osu! Game** is currently the biggest community driven, multi mode Discord server. By hosting events and encouraging interactions between their users, it's a good place to discuss all things related to gameplay. |
+| [osu! University](https://discord.gg/QubdHdnBVg) | [DigitalHypno](https://osu.ppy.sh/users/4384207) | **osu! University** is a community Discord server with a focus on discussion about improvement, mainly in the [osu!](/wiki/Game_mode/osu!) game mode. It regularly hosts various events such as top player interviews, [tournaments](/wiki/Tournaments), and improvement-related studies. |
+| [osu! Medal Hunters](https://discord.gg/osumedalhunters) | [MegaMix](https://osu.ppy.sh/users/18152711) | **osu! Medal Hunters** hosts discussion about [medals](/wiki/Medals) and [medal solutions](/wiki/Medals/Unlock_requirements). |
+| [osu!alternative](https://discord.gg/VZWRZZXcW4) | [billie eilish](https://osu.ppy.sh/users/6245906) | **osu!alternative** offers extra metrics and score trackers not provided by the website. It allows players to rank players and scores in an unofficial manner. |
+
+## Community development
+
+These servers focus on providing and developing tools and systems for the game. While the official development of osu! is done in the **osu!** Discord server, certain projects require their own workspaces before proposing and merging their work in game.
+
+| Name | Owner |
+| :-- | :-- |
+| [Performance Points](https://discord.gg/aqPCnXu) | - |
+| [osu!catch dev server](https://discord.gg/YEJBENvFzN) | [bastoo0](https://osu.ppy.sh/users/4864877) |
+| [o!m SR/PP Rework Hub](https://discord.gg/GFCNNg8bwk) | [Quenlla](https://osu.ppy.sh/users/4725379) |
+
+## Mapping and modding
+
+The **Modding & Mapping Hubs** are community Discord servers designed for all current and aspiring mappers and modders. These are places meant to interact between other users interested in mapping for the game mode. They offer a place to organise, discuss, and advertise different mapping projects.
+
+| Name | Owner |
+| :-- | :-- |
+| [osu!taiko Modding & Mapping Hub](https://discord.gg/yRjvvyZ) | [Hivie](https://osu.ppy.sh/users/14102976) |
+| [osu!catch Modding and Mapping Hub](https://discord.gg/ZuxFc4q) | [Ascendance](https://osu.ppy.sh/users/2931883) |
+| [osu!mania Mapping & Modding Hub](https://discord.gg/FqbDdYN) | [Maxus](https://osu.ppy.sh/users/4335785) |
+
+### Other
```

Comment:
this is mb for the suggestion earlier, i didnt realise how this section would end up being formatted. this header is unnecessary but everything else is good